source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,63 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      if (process.platform === 'darwin') {
+        if (this._tray == null) {
+          let basepath = app.getAppPath()
+          let image = await nativeImage.createThumbnailFromPath(basepath + '/source/main/assets/icons/black_white_128.png', { width: 16, height: 16 })
+          this._tray = new Tray(image)
+
+          // this._tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
+          const contextMenu = Menu.buildFromTemplate([
+            {
+              label: 'Show Zettlr',
+              click: () => {
+                this.showAnyWindow()
+              },
+              type: 'normal'
+            },
+            { label: '', type: 'separator' },
+            {
+              label: 'Quit',
+              click: () => {
+                app.quit()
```

Comment:
The tray isn't working for me so I can't test at the moment, but does `app.quit()` follow this part of the [tray design](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Tray-design)?:

> If there are unsaved changes, show and bring-to-the-front the Zettlr window and display the 'There are unsaved changes' modal dialog.

 I tried to follow the  Zettlr code and if I follow correctly this will quit immediately?